electron/electron.js

```diff
@@ -74,3 +74,25 @@ ipcMain.handle('get-file', async () =>
     properties: ['openFile'],
   }),
 );
+
+ipcMain.handle('get-output-path', async () => {
+  const options = {
+    title: 'Select Output Folder',
+    defaultPath: app.getPath('downloads'),
+    properties: ['openDirectory', 'createDirectory'],
+  };
+  return dialog.showOpenDialog(null, options, (savePath) => savePath);
```

Comment:
DP1 - Take a look at the dialog documentation to see if there is a better alternative to this function here. https://www.electronjs.org/docs/api/dialog

Reply:
What different features should I look for in an alternative?

Reply:
Well this is a dialog for opening a file. Explore the documentation to see if we can find one that's more applicable to what we're actually doing with that dialog, instead of opening a file.

Reply:
The other option is dialog.showSaveDialog, but unfortunately it turns out that showSaveDialog doesn't have the option of just selecting a folder--the dialog requires the user to input a file name to save as. Since the save process is supposed to save multiple files with predetermined names to a folder, the showSaveDialog function doesn't match with the purpose of the save process, and may confuse users. The dialog.showOpenDialog function has an option ('openDirectory') that lets you limit it so that the user can only select folders. 
Luckily it turns out there's an option to change the confirmation button text to something custom. I changed it from "Open" to "Save", so now showOpenDialog looks the same as showSaveDialog, only without the section to input a file name to save as. Is this an okay solution, or should I keep looking for another way?

Reply:
I don't think you've pushed that change yet, but if that looks okay to you I'd be fine with that for now – primarily because we're looking to implement a more sophisticated UI for saving all extracted data. Also let me know if you're still looking into a custom UI for this. If you've put that on pause, just let me know.

Reply:
Okay. I've pushed the button label change. I'm still looking into using a more sophisticated UI for this, or potentially a form inside of a showMessageBox.